app/renderer/components/messageBox.js

```diff
@@ -0,0 +1,165 @@
+    return this.props.tabId || ''
+  }
+
+  get title () {
+    const msgBoxTitle = (this.props.detail && this.props.detail.get('title')) || ''
+    return msgBoxTitle.replace(config.braveExtensionId, 'Brave')
+  }
+
+  get message () {
+    return (this.props.detail && this.props.detail.get('message')) || ''
+  }
+
+  get buttons () {
+    return (this.props.detail && this.props.detail.get('buttons')) || makeImmutable(['OK'])
+  }
+
+  get cancelId () {
+    return this.props.detail && this.props.detail.get('cancelId')
+  }
+
+  get suppress () {
+    return (this.props.detail && this.props.detail.get('suppress')) || false
+  }
+
+  get showSuppress () {
+    return (this.props.detail && this.props.detail.get('showSuppress')) || false
+  }
+
+  onClick (e) {
+    e.stopPropagation()
+  }
+
+  onKeyDown (e) {
+    if (this.props.isActive) {
+      switch (e.keyCode) {
+        case KeyCodes.ENTER:
+          this.onDismiss(this.tabId)
+          break
+        case KeyCodes.ESC:
+          this.onDismiss(this.tabId, this.cancelId)
+          break
+      }
+    }
+  }
+
+  onSuppressChanged (e) {
+    const detail = this.props.detail.toJS()
+    detail.suppress = !detail.suppress
+    appActions.tabMessageBoxUpdated(this.tabId, detail)
+  }
+
+  onDismiss (tabId, buttonId) {
+    const response = {
+      suppress: this.suppress,
+      result: true
+    }
+
+    if (typeof this.cancelId === 'number') {
+      response.result = buttonId !== this.cancelId
+    }
+
+    appActions.tabMessageBoxDismissed(this.tabId, response)
+  }
+
+  get messageBoxButtons () {
+    const buttons = []
+
+    for (let index = (this.buttons.size - 1); index > -1; index--) {
+      buttons.push(<Button l10nId={this.buttons.get(index)}
+        className={index === 0 ? 'primaryButton' : 'whiteButton'}
+        onClick={this.onDismiss.bind(this, this.tabId, index)} />)
+    }
+
+    return buttons
+  }
+
+  render () {
+    return <Dialog className='noOutline'>
+      <div data-test-id={'msgBoxTab_' + this.tabId}
+        onClick={this.onClick}
+        onKeyDown={this.onKeyDown}
+        className={css(
+          commonStyles.flyoutDialog,
+          styles.container
+        )}>
+        <div className={css(styles.title)} data-test-id='msgBoxTitle'>
+          {this.title}
+        </div>
+        <div className={css(styles.body)} data-test-id='msgBoxMessage'>
+          {this.message}
+        </div>
+        {
+          this.showSuppress
+            ? <SwitchControl
+              rightl10nId='preventMoreAlerts'
+              checkedOn={this.suppress}
+              onClick={this.onSuppressChanged} />
+            : null
+        }
+        <div className={css(styles.buttons)} data-test-id='msgBoxButtons'>
+          {this.messageBoxButtons}
+        </div>
+      </div>
+    </Dialog>
+  }
+}
+
+const styles = StyleSheet.create({
+  'container': {
+    outline: 'none',
+    display: 'block'
+  },
+  'title': {
+    fontWeight: 'bold',
+    fontSize: '12pt',
```

Comment:
Good question- I was thinking it would be nice to use official point sizes and then have everything else relative using em.  @NejcZdovc suggested using rem for font size.  We should work together and propose a standard cc: @luixxiul